src/librustc_mir/transform/check_unsafety.rs

```diff
@@ -190,12 +201,17 @@ impl<'a, 'tcx> Visitor<'tcx> for UnsafetyChecker<'a, 'tcx> {
                                 if elem_ty.moves_by_default(self.tcx, self.param_env,
                                                             self.source_info.span) {
                                     self.require_unsafe(
-                                        "assignment to non-`Copy` union field")
+                                        "assignment to non-`Copy` union field",
+                                        "the previous content of the field may be dropped, which \
+                                         cause undefined behavior if the field was not properly \
```

Comment:
Typo: "which cause"

Reply:
I must be blind, what is the typo there?

Reply:
Oh, it must be "which causes", of course. Sorry!